src/lsstseries/analysis/structurefunction2.py

```diff
@@ -1,17 +1,14 @@
 import numpy as np
 import pandas as pd
-from scipy.stats import binned_statistic
 
+from lsstseries.analysis.structure_function_calculators import SF_METHODS
 
-def calc_sf2(
-    lc_id, time, flux, err, band, bins=None, band_to_calc=None, combine=False, method="size", sthresh=100
-):
-    """Compute structure function squared on one or many bands
+
+def calc_sf2(time, flux, err=None, band=None, lc_id=None, sf_method="basic", argument_container=None):
+    """_summary_
```

Comment:
What's with the new empty summary thing?

Reply:
Gross oversight, fixed!